Lib/inspect.py

```diff
@@ -881,29 +881,28 @@ def cleandoc(doc):
 
     Any whitespace that can be uniformly removed from the second line
     onwards is removed."""
-    try:
-        lines = doc.expandtabs().split('\n')
-    except UnicodeError:
-        return None
-    else:
```

Comment:
I removed this try-except-else block because Python 3 don't autodecode from bytes.
If doc is bytes, `doc.split('\n')` raises TypeError, not UnicodeError.